src/main/java/com/jvm_bloggers/utils/HtmlEmptinessChecker.java

```diff
@@ -4,6 +4,10 @@
 
 public class HtmlEmptinessChecker {
 
+    private HtmlEmptinessChecker() {
+        throw new IllegalStateException("Utility class");
+    }
+
```

Comment:
We have a Lombok dependency in the project already. We can use [`@UtilityClass`](https://projectlombok.org/features/experimental/UtilityClass) annotation instead.
```suggestion
@UtilityClass
public class HtmlEmptinessChecker {
```

Reply:
Thank you. Done :)